mysql-test/t/percona_heap_bug_ps296.test

```diff
@@ -0,0 +1,36 @@
+
+
+--disable_query_log
```

Comment:
Why?

Reply:
I'll remove both this and the later disabling instructions - but keeping the result log disabled is probably a good idea here, because that stored procedure inserts a few thousands records.